cl/opinion_page/templates/opinions.html

```diff
@@ -332,10 +332,10 @@ <h4 class="case-court">{{ cluster.docket.court }}</h4>
                         {% endif %}
                     </li>
                 {% endif %}
-                {% if has_downloads and "pdf" in pdf_path %}
+                {% if has_downloads and "pdf" in pdf_path or cluster.filepath_pdf_harvard %}
                     <li role="presentation" {% if tab == "pdf" %} class="active" {% endif %}>
                         {% if tab == "pdf" %}
```

Comment:
Is this if statement just here to make it so the tab isn't a link when it's the active tab? I don't think that's really necessary and I think we can simplify by removing this if statement here (and presumably elsewhere)?

Reply:
its also to avoid displaying the PDF tab if the download is an RTF or HTML file or some other form that we are not currently prepared to display.

Reply:
Oh sorry, I meant line 337.

Reply:
yep